src/components/GapList/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+
+const spacingScale = (num) => {
+  switch (num) {
+    case 1:
+      return 4;
+    case 2:
+      return 8;
+    case 3:
+      return 16;
+    case 4:
+      return 24;
+    case 5:
+      return 32;
+    default:
+      return 16;
+  }
+};
```

Comment:
So.... yeah 😅 

- This is for applying the negative margin solution
- I can't use classes as we don't have negative margin variants (and we may only need them in FDS)
- There's really no way I know of for "dynamically" constructing a variable, outside of using `eval`.
- This is sort of one of the bad parts about using consistent jumps. If we had the jumps at 1, 2, 4, 6, and 7, then this function would just be multiplying.
- Let me know if you have any thoughts on this. We can easily make this a utility and use it where we need to, but yeah. Maybe the way here is to use css variables possibly....